docs/pages/desktop-access/getting-started.mdx

```diff
@@ -1,214 +1,171 @@
-</Details>
+<Admonition type="warning">
+Teleport Desktop Access is not yet compatible with Windows Server 2022.
+</Admonition>
 
 ## Prerequisites
 
-<ScopedBlock scope={["oss", "enterprise"]}>
-- One or more hosts to run the Teleport Auth and Proxy services on.
-</ScopedBlock>
-- A server or virtual machine running a Windows Server operating system.
-  In this guide, we'll install Active Directory on this server in order
-  to support passwordless logins with Teleport to the Windows desktops
-  in the Active Directory domain.
-- A Linux host where you will run the Teleport Desktop Service.
-  <ScopedBlock scope={["oss", "enterprise"]}> This guide assumes that you will
-  run Teleport's Windows Desktop Service on a dedicated host. To install Desktop
-  Access into an existing Teleport instance running other services, see the
-  [Manual Setup](manual-setup.mdx) guide for Desktop Access.</ScopedBlock>
-- An Active Directory domain, configured for LDAPS (Teleport requires an
-  encrypted LDAP connection). Typically this means installing
-  [AD CS](https://learn.microsoft.com/en-us/windows-server/identity/ad-cs/)
+- (!docs/pages/includes/edition-prereqs-tabs.mdx!)
 
-## Step 1/2. Install Teleport
+- A Linux server to run the Teleport Desktop Access service on.
+  You can reuse an existing server running any other Teleport instance.
 
-### Set up the Teleport Auth and Proxy Services
+- (!docs/pages/includes/tctl.mdx!)
 
-(!docs/pages/includes/database-access/start-auth-proxy.mdx!)
+## Step 1/4. Prepare Windows
 
-## Step 2/2. Run the discovery wizard
+### Import the Teleport root certificate
 
-In your web browser, access the teleport Web UI at <ScopedBlock scope={["oss", "enterprise"]}>
-`teleport.example.com`</ScopedBlock><ScopedBlock scope={["cloud"]}>
-`mytennant.teleport.sh`</ScopedBlock>. Click on your user name at the top right
-and select **Manage Access**, Select "Desktop" from the main menu, then **NEXT**:
+Use `tctl` to export the Teleport root certificate:
 
-If you already have Active Directory installed, skip to the next step. Otherwise,
-copy and paste the first command provided into a Windows PowerShell window. If
-you aren't already running AD Certificate services, copy and paste the second
-command after the first one completes and the server restarts:
+```code
+$ tctl auth export --type=windows > teleport.cer
+```
 
-![Install Active Directory](../../img/desktop-access/install-active-directory.png)
+Copy this certificate to your Windows system, if you didn't run `tctl` from there.
 
-Once the server is restarted from one or both command executions, click **NEXT**.
+### Install the Teleport service for Windows
 
-Copy and paste the provided command into a Windows PowerShell window to download
-and run the 'configure Active Directory' script:
+From the Windows system, download the [Teleport service installer](#LinkToCome) {/*Rename as needed */}
+and run it. When prompted, select the Teleport certificate file from the previous step.
```

Comment:
There's also a headless install option that might be worth documenting here.

Reply:
I'd be happy to document it once I learn about it myself.

Reply:
You should be able to invoke the same installer from the command line with a `--help` flag to see the options.

Reply:
@probakowski please advise ☝️

Reply:
It turns out it will show correct output if you run it (with `--help`) from elevated prompt. If you do it from normal prompt it will show UAC screen and then start new elevated console which will close immediately as the program exits (same thing will happen if you create shortcut with `--help` parameter and run it by double click)

Reply:
I've added a tip below.